### ISSUE TITLE
Archived Workflows page has right pane CSS issue

### ISSUE THREAD

**Issue by aldeed** [argoproj/argo-workflows#9701]
### Pre-requisites

- [X] I have double-checked my configuration
- [X] I can confirm the issues exists when I tested with `:latest`
- [ ] I'd like to contribute the fix myself (see [contributing guide](https://github.com/argoproj/argo-workflows/blob/master/docs/CONTRIBUTING.md))

### What happened/what you expected to happen?

Starting with 3.4.0, the Archived Workflows page has a display bug that pushes the right pane down. It makes it really hard to get to the Logs button and such at the bottom.

![Screen Shot 2022-09-27 at 7 42 51 PM](https://user-images.githubusercontent.com/3012067/192662973-36375950-e34b-430d-b19e-8f4edb515124.png)

The only workaround I've found is to click the zoom out button a bunch to make the node tree smaller, which then makes the right pane taller.

You can see it with any archived workflow, but it's most obvious when the tree is vertical, has a lot of nodes, and/or you zoom in on the tree to make it bigger (i.e., the size of the tree seems to be what pushes the right pane down).

### Version

3.4.0

### Paste a small workflow that reproduces the issue. We must be able to run the workflow; don't enter a workflows that uses private images.

```YAML
apiVersion: argoproj.io/v1alpha1
kind: Workflow
metadata:
  generateName: hello-world-
  labels:
    workflows.argoproj.io/archive-strategy: "always"
  annotations:
    workflows.argoproj.io/description: |
      This is a simple hello world example.
      You can also run it in Python: https://couler-proj.github.io/couler/examples/#hello-world
spec:
  entrypoint: whalesay
  templates:
  - name: whalesay
    container:
      image: docker/whalesay:latest
      command: [cowsay]
      args: ["hello world"]
```

### Logs from the workflow controller

not relevant?

### Logs from in your workflow's wait container

not relevant?

**Comment by terrytangyuan** [comment]
@dakotalillie @momom-i This might be related to one of your changes. Would you like to help take a quick look?

**Comment by aldeed** [comment]
Forgot to mention that the regular workflows page does not have this problem

**Comment by tooptoop4** [comment]
https://github.com/argoproj/argo-workflows/pull/9634 fixes this but there hasn't been a new release with it yet

**Comment by terrytangyuan** [comment]
I believe @aldeed is using a custom release based on master branch that already includes #9634.

**Comment by momom-i** [comment]
> This might be related to one of your changes. Would you like to help take a quick look?
It was a long time ago that I contributed, so I'll try to remember and look at it when I have time!

**Comment by aldeed** [comment]
@terrytangyuan We are using the custom patch for the workflow controller image but probably not the UI/server. I think that is just using released 3.4.0.

**Comment by terrytangyuan** [comment]
Could you use the same tag for Argo-server and see if the problem persists?

**Comment by dakotalillie** [comment]
Sorry for not seeing this earlier (and for introducing this bug in the first place).I can confirm that #9634 should resolve this issue. Here's a screencap using the latest code from master:

<img width="2048" alt="Screen Shot 2022-09-29 at 2 00 48 PM" src="https://user-images.githubusercontent.com/20373585/193141144-a135cb5e-e79c-4a74-988f-87e3d6c41eb0.png">

**Comment by sarabala1979** [comment]
@aldeed can you try on v3.4.1?

**Comment by jmeridth** [comment]
Confirmed this is fixed in v3.4.1 for us.
